pkg/cmd/server/origin/auth.go

```diff
@@ -375,8 +389,17 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 				}
 				passwordSuccessHandler := handlers.AuthenticationSuccessHandlers{c.SessionAuth, redirectSuccessHandler{}}
 
+				loginPath := OpenShiftLoginPrefix
```

Comment:
you can also remove the log message `fmt.Sprintf("Started Login endpoint at %%s%s", OpenShiftLoginPrefix),`... hitting that URL directly wouldn't help anyone, since they'd have no destination redirect